data-prepper-plugins/prometheus-sink/src/main/java/org/opensearch/dataprepper/plugins/sink/configuration/ThresholdOptions.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.sink.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import org.hibernate.validator.constraints.time.DurationMax;
+import org.hibernate.validator.constraints.time.DurationMin;
+import org.opensearch.dataprepper.model.types.ByteCount;
+
+import java.time.Duration;
+
+/**
+ * An implementation class of http index configuration Options
+ */
+public class ThresholdOptions {
+
+    private static final String DEFAULT_BYTE_CAPACITY = "50mb";
+
+    @JsonProperty("event_count")
+    @Size(min = 0, max = 10000000, message = "event_count size should be between 0 and 10000000")
+    @NotNull
+    private int eventCount;
+
+    @JsonProperty("maximum_size")
+    private String maximumSize = DEFAULT_BYTE_CAPACITY;
```

Comment:
Please use the `ByteCount` type here.

See for example: https://github.com/opensearch-project/data-prepper/blob/f8091635d2c60bdc3f9b17a9033b4c67e0fc739c/data-prepper-plugins/s3-sink/src/main/java/org/opensearch/dataprepper/plugins/sink/s3/configuration/ThresholdOptions.java#L49-L51

Reply:
Thanks for the review @dlvenable , ThresholdOptions class is already same like s3-sink